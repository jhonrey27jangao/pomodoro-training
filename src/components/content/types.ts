export type ContentType = {
  children?: any;
  title?: string;
};
