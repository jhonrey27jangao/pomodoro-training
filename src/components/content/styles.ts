import { ContentType } from "./types";
import styled from "styled-components";

export const StyledContent = styled.div<ContentType>`
  position: relative;
  margin-left: 250px;
  padding: 70px;
  font-family: "Heebo", sans-serif;

  .title-page {
    position: fixed;
    top: 40px;
    left: 20%;
    font-size: 30px;
    font-weight: bold;
  }
`;
