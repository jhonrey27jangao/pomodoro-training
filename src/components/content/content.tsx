import React from "react";
import { ContentType } from "./types";
import { StyledContent } from "./styles";

export const Content: React.FC<ContentType> = ({ title, children }) => (
  <StyledContent title={title}>
    <h1 className="title-page">{title}</h1>
    {children}
  </StyledContent>
);
