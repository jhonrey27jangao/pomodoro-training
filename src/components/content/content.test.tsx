import React from "react";
import { render } from "@testing-library/react";
import { Content } from "./content";

describe("Test for Content Component", () => {
  it("should render Content", () => {
    const { container } = render(<Content />);
    expect(container).toMatchSnapshot();
  });
});
