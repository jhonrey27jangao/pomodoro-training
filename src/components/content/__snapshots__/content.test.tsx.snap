// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Content Component should render Content 1`] = `
<div>
  <div
    class="sc-AxjAm hiZnOP"
  >
    <h1
      class="title-page"
    />
  </div>
</div>
`;

exports[`Test for Section Component should render Section 1`] = `
<div>
  <div
    class="sc-AxjAm hiZnOP"
  >
    <h1
      class="title-page"
    />
  </div>
</div>
`;
